.rustfmt.toml

```diff
@@ -10,15 +10,46 @@ reorder_imports = true
 reorder_modules = true
 tab_spaces = 4
 use_field_init_shorthand = true
+use_small_heuristics = "Default"
 use_try_shorthand = true
-edition = "2018"
 
-# unstable options (for one-off fixes and CI)
-merge_imports = true
-wrap_comments = true
+# unstable options
+binop_separator = "Front"
+blank_lines_lower_bound = 0
+blank_lines_upper_bound = 1
+brace_style = "PreferSameLine"
```

Comment:
`SameLineWhere` is the default value
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#brace_style

The only difference between this new value and the default one is whether the brace after a `where` clause is on the same line or the next line.  I thought the same line looked better, and more consistent with the other uses of opening braces, but it's not something I feel strongly about.

Reply:
I prefer `SameLineWhere`

Reply:
I'll change it.